packages/pm2-launcher/bin/prod

```diff
@@ -1 +1 @@
-NODE_ENV=production ./node_modules/pm2/bin/pm2-runtime start ./node_modules/@greenpress/pm2-launcher/greenpress-prod.config.js --env production
+NODE_ENV=production pm2-runtime start ./node_modules/@greenpress/pm2-launcher/greenpress-prod.config.js --env production
```

Comment:
didn't adding the full path to pm2-runtime solved our previous issue with the docker, or was it completely because of the mismatch between our version of docker-compose?

Reply:
It can by fixed after we install pm2 globally..